server/core/controller.js

```diff
@@ -19,6 +45,10 @@ class Controller {
 
             sortField[req.query.sort] = (req.query.sortType) ? req.query.sortType : 1;
 
+            var int = Number(sortField[req.query.sort]);
+
+            sortField[req.query.sort] = int;
```

Comment:
You can merge those two lines into:

```js
sortField[req.query.sort] = Number(sortField[req.query.sort]);
```

It is still readable and avoids having a badly named temporary variable.